deployment/artifacts/mainnet.json

```diff
@@ -6703,9 +6716,9 @@
                     "outputs": []
                 }
             ],
-            "tx_hash": "0x6eeafdcd35f18a83e9ba17ba67483b56ae21a5611b53f9b11da026ac073b5558",
-            "block_number": 50223997,
-            "deployer": "0xFfFd7092685bDeeBD121D1A0FEA3c349114Cce50"
+            "tx_hash": "0x31474ff61017606ecb0ae534778b7eed76b357cc631c7c7e99ce2237ab9f3b50",
+            "block_number": 75720593,
+            "deployer": "0x1591165F1BF8B73de7053A6BE6f239BC15076879"
```

Comment:
I thought these lines shouldn't change, just the ABI?

These lines track the original deployment of the proxy, not the implementation. Or am I mistaken? That being said, I don't know if our registry merge script handles that properly.

Reply:
I'm not sure, I thought it was the opposite – that they were tracking the latest implementation. Either way, currently it's a just a convention, so we should think what we want and codify it as part of the merge script.